src/process/index.html

```diff
@@ -25,14 +25,29 @@
     ],
     "steps": [
       {
-        "title": "Clarify your goals",
+        "title": "1. Clarify your goals",
         "description": "You're interested in buying a new home -- you wouldn't be here if you weren't. However, it's a good idea to take a minute to think about why you want to buy. What are your needs, motivations, and priorities? How will buying a new home fit with your lifestyle needs?",
         "key_tool": { 
           "text": "Why buy? Why now?",
           "url": "/" 
         },
-        "how_to_take_action": "<p>html goes here</p>",
-        "what_to_know": "<p>html goes here</p>"
+        "how_to_take_action": "<h5 class='h4 u-mb10'>Clarify your goals for buying a new home.</h5>
+        <p>We’ve developed a step-by-step worksheet for new homebuyers that will guide you through all the items in this action step.</p>
+        <h5 class='h4 u-mb10'>If possible, call your servicer before you miss a payment.</h5>
+        <p>If you’re already behind on your payments, contact your servicer right away.</p>
+        <ul class='list__branded list__flush-sides'>
+          <li class='list_item__spaced'>
+            Ask your servicer what options are available to help you get back on track or make it through temporary financial difficulties.
+          </li>
+          <li class='checked list_item__spaced'>
+            Ask your servicer to send you a written explanation of options that may be available to you if you can’t make your payments in the future.    
+          </li>
+          <li class='alert list_item__spaced'>
+            If you are having problems with your servicer, you can submit a complaint with the CFPB.
```

Comment:
If these patterns are consistent across all the steps, I think we can probably apply most of the styling automatically so users only have to add the `.alert` & `.checked` classes in WordPress.